lib/plugins/stonith/apcmastersnmp.c

```diff
@@ -246,7 +246,7 @@ APC_open(char *hostname, int port, char *community)
     /* fill session */
     session.peername = hostname;
     session.version = SNMP_VERSION_1;
-    session.remote_port = port;
+    /* session.remote_port = port; */
```

Comment:
Why is setting the port dropped?

Reply:
It's been a while. I think the comments are leftovers from debugging. They should not have been in the PR, I guess.

Reply:
net-snmp "session remote_port" is documented as UNUSED and deprecated since decades (at least since before 2004, I suspect since 2000). It only "recently" (2017) was given deprecated annotation in the header files, and even more recently those updated header files made it into distributions.
And along with gcc -Wdeprecated-declarations and -Werror, that probably failed your build at some point.

Given that obviously no-one has used any non-default port (at least not successfully...) commenting out the assignment is the minimal fix. I'll probably add a commit doing just that.

Reply:
Ah yes, that's why! Now I (very faintly) remember. 

Thanks for adding this, I should have added it to the commit message back then.